gammapy/stats/fit_statistics_jit.py

```diff
@@ -0,0 +1,144 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+from numba import jit
+
+global TRUNCATION_VALUE  # compile time constant
+TRUNCATION_VALUE = 1e-25
```

Comment:
Can we avoid declaring the `TRUNCATION_VALUE` twice? Can it be imported from fit_statistic_cython? OR conversely imported there from here?